package.json

```diff
@@ -16,8 +16,11 @@
     "precommit": "lint-staged",
     "prepare": "patch-package",
     "prettier-write": "yarn run prettier --write",
+    "relay": "node scripts/relayCompiler.js",
     "sentry": "sh scripts/sentry.sh",
     "start": "sh scripts/start.sh",
+    "start:relay": "yarn relay && concurrently --raw --kill-others 'yarn relay --watch' 'yarn start'",
+    "sync-schema": "rm -rf data && mkdir data && graphql-fetch-schema https://metaphysics-staging.artsy.net -o data",
```

Comment:
@alloy @orta - what do ya'll think about this slick `graphql-fetch-schema` lib? Any issue? Seems like this is a nicer approach than having to use submodules.

Reply:
I agree on the nicer approach ( [I use it in Peril](https://github.com/danger/peril.systems/blob/master/package.json#L15-L16) ) - I plan to use it in our setup for https://github.com/artsy/peril-settings/issues/51 so maybe we could consolidate them on [graphql-cli](https://www.npmjs.com/package/graphql-cli)?

Reply:
Nice 👍